src/components/TitleInput/TitleInput.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+type Props = {
+  todoTitle: string;
+  handleInputChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  hasTitleError: boolean;
+};
+
+export const TitleInput: React.FC<Props> = ({
```

Comment:
better rename it to just FormInput or TextField or something like this, with such approach you could easier reuse it